scenarios/throttle_test.erl

```diff
@@ -1,119 +0,0 @@
-%%%-------------------------------------------------------------------
```

Comment:
this one should stay. it's just a general amoc_throttle example. it doesn't require MIM and can be started on just pure amoc